wicket-core/src/main/resources/META-INF/wicket.xsd

```diff
@@ -102,6 +103,15 @@
 		</simpleType>
 	</attribute>
 
+	<attribute name="auto" type="boolean">
+		<annotation>
+			<documentation>Can be used in HTML <![CDATA[<label]]> to indicate the label will be automatically updated
```

Comment:
```suggestion
			<documentation>The attribute can be used in HTML &lt;label&gt; to indicate the label will be automatically updated
```

Reply:
Hum... other elements use CDATA

![Uploading image.png…]()

Maybe change all of them too?

Reply:
![Screenshot from 2024-03-28 15-48-36](https://github.com/apache/wicket/assets/462655/4a241206-3341-4b76-8fae-655a1efc913f)

Reply:
I updated the suggestion